runtime/j9vm/jvm.c

```diff
@@ -4474,12 +4557,20 @@ JVM_RaiseSignal(jint sigNum)
 {
 	jboolean rc = JNI_FALSE;
 	J9JavaVM *javaVM = (J9JavaVM *)BFUjavaVM;
+	BOOLEAN isShutdownSignal = isSignalUsedForShutdown(sigNum);
```

Comment:
Is isSignalUsedForShutdown() misnamed? Should it be called isAsyncSignal() or similar?

Reply:
this function is supposed to determine if a signal is used by the JVM for shutdown. Referring to [Terminator.java Unix](https://github.com/ibmruntimes/openj9-openjdk-jdk9/blob/openj9/jdk/src/java.base/unix/classes/java/lang/Terminator.java#L59) and [Terminator.java Windows](https://github.com/ibmruntimes/openj9-openjdk-jdk9/blob/openj9/jdk/src/java.base/windows/classes/java/lang/Terminator.java#L59), a shutdown handler is registered for the following signals: `SIGINT`, `SIGTERM` and `SIGHUP`. this function checks if a signal is either `SIGINT`, `SIGTERM` or `SIGHUP`. this function is needed because the spec provided by the JCL team puts special constraints on these shutdown signals.